main.go

```diff
@@ -565,6 +565,10 @@ func mainCore() error {
 	webMux.Get("/search", explore.Search)
 	webMux.Get("/charts", explore.Charts)
 	webMux.Get("/ticketpool", explore.Ticketpool)
+	webMux.Get("/statistics", explore.StatsPage)
+	webMux.Get("/stats", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I would prefer to do this the other way.  (`/statistics` -> `/stats`).  Keeping the URL short is preferable.